From ed6db1e1583be873271389befa61a6f238701d94 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ball=C3=B3=20Gy=C3=B6rgy?= <ballogyor@gmail.com>
Date: Thu, 23 Oct 2025 19:35:57 +0200
Subject: [PATCH] Remove deprecated properties

This fixes the following theme parsing errors using GTK 3.24:
```
Gtk-WARNING **: 19:33:09.442: Theme parsing error: gtk-widgets.css:16:32: The style property GtkExpander:expander-size is deprecated and shouldn't be used anymore. It will be removed in a future version

Gtk-WARNING **: 19:33:09.442: Theme parsing error: gtk-widgets.css:17:35: The style property GtkExpander:expander-spacing is deprecated and shouldn't be used anymore. It will be removed in a future version

Gtk-WARNING **: 19:33:09.442: Theme parsing error: gtk-widgets.css:23:47: The style property GtkProgressBar:min-horizontal-bar-height is deprecated and shouldn't be used anymore. It will be removed in a future version

Gtk-WARNING **: 19:33:09.442: Theme parsing error: gtk-widgets.css:24:44: The style property GtkProgressBar:min-vertical-bar-width is deprecated and shouldn't be used anymore. It will be removed in a future version
```
---
 gtk3/theme/3.20/gtk-widgets.css.em | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/gtk3/theme/3.20/gtk-widgets.css.em b/gtk3/theme/3.20/gtk-widgets.css.em
index 0b953a5..8bd8d6d 100644
--- a/gtk3/theme/3.20/gtk-widgets.css.em
+++ b/gtk3/theme/3.20/gtk-widgets.css.em
@@ -66,16 +66,10 @@ disabled_opacity = 0.5
 
     -GtkScrolledWindow-scrollbar-spacing: 0;
 
-    -GtkExpander-expander-size: $font_height;
-    -GtkExpander-expander-spacing: 2;
-
     -GtkTreeView-expander-size: $font_height;
 
     -GtkArrow-arrow-size: 1.0;
 
-    -GtkProgressBar-min-horizontal-bar-height: $subcell_size;
-    -GtkProgressBar-min-vertical-bar-width: $subcell_size;
-
     -GtkWidget-text-handle-width: 110px;
     -GtkWidget-text-handle-height: 110px;
 }
